src/db.c

```diff
@@ -2167,7 +2179,7 @@ int xreadGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysResult
     keys = getKeysPrepareResult(result, num);
     for (i = streams_pos+1; i < argc-num; i++) {
         keys[i-streams_pos-1].pos = i;
-        keys[i-streams_pos-1].flags = CMD_KEY_READ; 
+        keys[i-streams_pos-1].flags = 0; 
```

Comment:
maybe add a top comment for `xreadGetKeys` about the fact it doesn't maintain flags and why